conftest.py

```diff
@@ -22,3 +14,33 @@ def mock_inline_css(monkeypatch):
     """
     import amo.helpers
     monkeypatch.setattr(amo.helpers, 'is_external', lambda css: True)
+
+
+def prefix_indexes(config):
+    """Prefix all ES index names and cache keys with `test_` and, if running
+    under xdist, the ID of the current slave."""
+
+    if hasattr(config, 'slaveinput'):
+        prefix = 'test_{[slaveid]}'.format(config.slaveinput)
+    else:
+        prefix = 'test'
+
+    from django.conf import settings
+
+    # Ideally, this should be a session-scoped fixture that gets injected into
+    # any test that requires ES. This would be especially useful, as it would
+    # allow xdist to transparently group all ES tests into a single process.
+    # Unfurtunately, it's surprisingly difficult to achieve with our current
+    # unittest-based setup.
+
+    for key, index in settings.ES_INDEXES.items():
+        if not index.startswith(prefix):
+            settings.ES_INDEXES[key] = '{prefix}_amo_{index}'.format(
+                prefix=prefix, index=index)
+
+    settings.CACHE_PREFIX = 'amo:{0}:'.format(prefix)
+    settings.KEY_PREFIX = settings.CACHE_PREFIX
```

Comment:
I have the feeling this setting is unused (or rather, that it's just the same as `CACHE_PREFIX` everywhere, but that's something to be checked/fixed in another PR if needed ;)